src/main/scala/swiss/dasch/domain/Asset.scala

```diff
@@ -0,0 +1,29 @@
+package swiss.dasch.domain
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.types.string.NonEmptyString
+import swiss.dasch.config.Configuration.StorageConfig
+import zio.*
+import zio.nio.file.{ Files, Path }
+
+import java.io.IOException
+
+opaque type ProjectShortcode = NonEmptyString
+type IiifPrefix              = ProjectShortcode
+
+trait AssetService {
```

Comment:
Shouldn't filename be exact as the trait?

Reply:
That would be Java.